src/internal/functions/Get-TypeParameterContent.ps1

```diff
@@ -0,0 +1,54 @@
+Function Get-TypeParameterContent {
+  <#
+  .SYNOPSIS
+    Return the text for a Puppet Resource API Type given a DSC Resouce.
```

Comment:
This synopsis and description are exactly the same as for the Get-TypeContent function.